src/structures/ClientUser.js

```diff
@@ -297,6 +297,14 @@ class ClientUser extends User {
       );
     }
   }
+
+  /**
+   * @param {string} code Invite code
+   * @returns {Promise<Guild>} Joined guild
+   */
+  acceptInvite(code) {
```

Comment:
This should probably also be able to take an `Invite` object that could have been previously cached